resources/puppetlabs/lein-ezbake/template/global/ext/redhat/ezbake.service.erb

```diff
@@ -10,7 +10,12 @@
 #
 [Unit]
 Description=<%= EZBake::Config[:project] %> Service
-After=syslog.target network.target
+
+<% if EZBake::Config[:build_type] == 'foss' -%>
```

Comment:
Would it make sense to have a function that detects if we are building for foss instead of putting that logic in erb templates?